equistore-core/src/labels.rs

```diff
@@ -332,3 +342,17 @@ impl std::ops::Index<usize> for Labels {
         &self.values[start..stop]
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn valid_names() {
+        let e = LabelsBuilder::new(vec!["not an ident"]).err().unwrap();
+        assert_eq!(e.to_string(), "invalid parameter: all labels names must be valid identifiers, 'not an ident' is not");
+
+        let e = LabelsBuilder::new(vec!["not", "there", "not"]).err().unwrap();
+        assert_eq!(e.to_string(), "invalid parameter: labels names must be unique, got 'not' multiple times");
```

Comment:
Is this definitely tested correctly?

When I try:
```
Labels(names=["not", "there", "not"], values=np.array([[0, 1, 2],]))
```
with this build I get a different error:
`ValueError: field 'not' occurs more than once`